From 5db22588d3128be7c02c5479405f12eb4dcc8d57 Mon Sep 17 00:00:00 2001
From: Xiaohan Wang <xhwang@chromium.org>
Date: Thu, 7 Jul 2022 20:02:05 +0000
Subject: [PATCH 069/110] sheriff: Disable
 MediaEngagementContentsObserverFencedFrameBrowserTest.SendEngagementLevelToRenderFrameOnFencedFrame

This test is flaky on multiple platforms.

(cherry picked from commit 754c401123e0cab7aeed5abddda74b41821482c3)

Bug: 1320380
Test: This CL disables flaky/failing test(s)
Change-Id: Id2ba2aca193957df68000200c7848c9ba5fdba64
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3611484
Auto-Submit: Xiaohan Wang <xhwang@chromium.org>
Commit-Queue: Francois Pierre Doray <fdoray@chromium.org>
Commit-Queue: Xiaohan Wang <xhwang@chromium.org>
Reviewed-by: Francois Pierre Doray <fdoray@chromium.org>
Owners-Override: Francois Pierre Doray <fdoray@chromium.org>
Cr-Original-Commit-Position: refs/heads/main@{#996778}
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3750739
Commit-Queue: Robert Liao <robliao@chromium.org>
Auto-Submit: Robert Liao <robliao@chromium.org>
Reviewed-by: Xiaohan Wang <xhwang@chromium.org>
Owners-Override: Robert Liao <robliao@google.com>
Cr-Commit-Position: refs/branch-heads/5005@{#1228}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 chrome/browser/media/media_engagement_browsertest.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/chrome/browser/media/media_engagement_browsertest.cc b/chrome/browser/media/media_engagement_browsertest.cc
index f370c75702de..5a515f851c0c 100644
--- a/chrome/browser/media/media_engagement_browsertest.cc
+++ b/chrome/browser/media/media_engagement_browsertest.cc
@@ -1040,8 +1040,9 @@ class MediaEngagementContentsObserverFencedFrameBrowserTest
   std::unique_ptr<content::test::FencedFrameTestHelper> fenced_frame_helper_;
 };
 
+// TODO(crbug.com/1320380): Re-enable this test
 IN_PROC_BROWSER_TEST_F(MediaEngagementContentsObserverFencedFrameBrowserTest,
-                       SendEngagementLevelToRenderFrameOnFencedFrame) {
+                       DISABLED_SendEngagementLevelToRenderFrameOnFencedFrame) {
   ASSERT_TRUE(embedded_test_server()->Start());
 
   MockAutoplayConfigurationClient client;
-- 
2.38.1

